### PR TITLE
Regression test for #871

### PR DESCRIPTION
**Pull request by drew-gross**

### CODE REVIEW

**Comment by flovilmart** [comment]
Syntax error in your test

File: spec/ParseRelation.spec.js [hunk 1]
```diff
@@ -652,4 +652,29 @@ describe('Parse.Relation testing', () => {
       }));
     });
   });
+
+  it('relations are not bidirectional (regression test for #871)', done => {
+    let PersonObject = Parse.Object.extend("Person");
+    let p1 = new PersonObject();
+    let p2 = new PersonObject();
+    Parse.Object.saveAll([p1, p2]).then(([p1, p2]) => {
```
Comment (flovilmart) [inline spec/ParseRelation.spec.js]: Causing a syntax error here

Comment (drew-gross) [inline spec/ParseRelation.spec.js]: Probably passed locally because I'm on a newer node. This is why we have CI :p

Comment (flovilmart) [inline spec/ParseRelation.spec.js]: Yeah.... I get that often too. NVM FTW

**Pull request merged**